examples/shadertoy_buffer_lovers.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+from wgpu_shadertoy import Shadertoy
+
+# shadertoy source: https://www.shadertoy.com/view/ssjyWc by FabriceNeyret2 (CC-BY-NC-SA-3.0?)
+
+# current "bug": the string kinda floats off to the upper right corner, without any inputs... ? Likely to be some issue with the implementation of buffers.
```

Comment:
Still applicable?

Reply:
still happens with the wgpu22.1 branch of wgpu-py 

It could be a variety of causes, so I am not even shure if it's something with this implementation.
We could search for a example that seems to be working as expected. I wanted to have one example that includes multiple buffers and complex interactions between them.

Reply:
so... it seems to have been an issue with the resolution. As in this particular shader doesn't like odd resolutions or some aspect ratios. It seems to behave well with the default 800x450 that is used by the website on 1080p screens. (although 1920x1080 full screen doesn't work).